pkg/server/container_create.go

```diff
@@ -42,7 +42,8 @@ func init() {
 
 // setOCIProcessArgs sets process args. It returns error if the final arg list
 // is empty.
-func setOCIProcessArgs(g *generator, config *runtime.ContainerConfig, image *imagespec.Image) error {
+func setOCIProcessArgs(g *generator, config *runtime.ContainerConfig, imagestoreImage *imagestore.Image) error {
```

Comment:
Maybe a name like `criImage` would be more natural here? `imagestoreImage` sounds a bit awkward.